installation/install.sh

```
@@ -3212,9 +3218,14 @@ cs_deploy_ui_db(){
 sz_start_zstack(){
     echo_subtitle "Start ${PRODUCT_NAME} management node (takes a couple of minutes)"
     trap 'traplogger $LINENO "$BASH_COMMAND" $?'  DEBUG
+    # Temporarily disable foreign_key_checks for MariaDB 10.6+ to workaround SNS initialization bug
+    mysql -u root -p${MYSQL_NEW_ROOT_PASSWORD} -e "SET GLOBAL foreign_key_checks=0;" >>$ZSTACK_INSTALL_LOG 2>&1
```

Shellcheck (0.11.0)

[style] Consider using { cmd1; cmd2; } >> file instead of individual redirects.

(SC2129)